### ISSUE TITLE
ATAC-seq pipeline exit if no mitochondrial reads are duplicated 

### ISSUE THREAD

**Issue by cschmidl** [epigen/open_pipelines#14]
ATAC-seq pipeline exit if no mitochondrial reads are duplicated due to zero division error - see logfile:
File "/Users/christianschmidl/src/open_pipelines/pipelines/atacseq.py", line 359, in parse_duplicate_stats
    prefix + "duplicate_percentage": (float(duplicates) / (single_ends + paired_ends * 2)) * 100}
ZeroDivisionError: float division by zero

**Comment by nsheff** [comment]
You should try [pepatac](https://github.com/databio/pepatac) -- I don't know that anyone is updating this ATAC-seq pipeline anymore... @afrendeiro ?

**Comment by afrendeiro** [comment]
Yes, I am maintaining it. But you can always try [pepatac](https://github.com/databio/pepatac).

**Comment by nsheff** [comment]
oh cool. any interest in combining with pepatac?
